model/image.go

```diff
@@ -337,6 +358,49 @@ func (img *Image) ToGoImage() (goimage.Image, error) {
 	return imgout, nil
 }
 
+// ToBinaryImage converts an image into binary (bi-level) image. Binary images
```

Comment:
Maybe we should call this ToBinary or ConvertToBinary instead of ToBinaryImage. Adding another Image in there seems redundant. I like `ConvertToBinary` better as it seems to convey that the conversion is done in place, instead of returning a new instance.

Reply:
Done.